### PR TITLE
feat(Songsterr): add presence

### PR DESCRIPTION
**Pull request by Weemizo**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details> 
image
image
image

<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Review by theusaf — COMMENTED** [review]
* Conflicts with #7385.

File: websites/S/Songsterr/presence.js
[the remark this answers is not in the document]
Comment (Weemizo) [reply websites/S/Songsterr/presence.js:1]: right, the wrong file i guess

File: websites/S/Songsterr/presence.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		largeImageKey: "https://i.imgur.com/vKeC7jt.png",
+		startTimestamp: browsingTimestamp,
+	};
+
+	if (document.location.search.startsWith("?pattern=")) {
+		const searchvar = (<HTMLInputElement>(
+			document.querySelector("#sticky-list-header > div.Ccm27v > div > input")
+		)).value;
+		presenceData.details = `Searching tabs for '${searchvar}'`;
+	} else if (document.location.pathname.endsWith("plus"))
```
Comment (theusaf) [inline websites/S/Songsterr/presence.ts:17]: Destructure `document.location` properties.

File: websites/S/Songsterr/presence.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	else if (document.location.pathname.endsWith("howtoreadtab"))
+		presenceData.details = "Learning how to read a tab";
+	else if (document.location.pathname.endsWith("account"))
+		presenceData.details = "Typing in account details";
+	else if (document.location.pathname.endsWith("favorites"))
+		presenceData.details = "Checking saved favorite songs";
+	else if (document.location.pathname.endsWith("payment"))
+		presenceData.details = "Buying Songsterr Plus";
+	else if (
+		(<HTMLInputElement>document.querySelector("#revisions-toggle")).title ===
```
Comment (theusaf) [inline websites/S/Songsterr/presence.ts:34]:
```suggestion
		(document.querySelector<HTMLInputElement>("#revisions-toggle")).title ===
```

File: websites/S/Songsterr/presence.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		"Hide revisions"
+	) {
+		presenceData.details = "Checking Revisions";
+		if (
+			(<HTMLInputElement>document.querySelector("#submitRevisionButton"))
```
Comment (theusaf) [inline websites/S/Songsterr/presence.ts:39]:
```suggestion
			(document.querySelector<HTMLInputElement>("#submitRevisionButton"))
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by theusaf** [comment]
Closing because of an existing PR for this website.

**Pull request closed without merging**